Makefile

```diff
@@ -46,6 +46,10 @@ it-stress: build
 	@echo "  >  \033[32mRunning Integration Tests stress mode...\033[0m "
 	GOSSAMER_NODE_HOST=0.0.0.0 GOSSAMER_INTEGRATION_TEST_MODE=stress go test ./tests/stress/... -timeout=5m -p 1 -short -v
 
+it-rpc: build
+	@echo "  >  \033[32mRunning Integration Tests RPC Specs mode...\033[0m "
+	GOSSAMER_NODE_HOST=0.0.0.0 GOSSAMER_INTEGRATION_TEST_MODE=rpc_spec go test ./tests/rpc/... -timeout=5m -p 1 -short -v
```

Comment:
`GOSSAMER_INTEGRATION_TEST_MODE=rpc_spec`

should be 

`GOSSAMER_INTEGRATION_TEST_MODE=rpc`

Reply:
So since the objective of naming it spec was to make a test suite, and you are not ok with calling it spec, but I really nice need to be something more than just rpc (since its supposed to be a test suite for the RPC endpoints that match against a test specification) I renamed the logs to RPC suite. Let me know if you have any objections.